src/install/install.md

```diff
@@ -19,29 +19,33 @@ Lute can be installed from [https://pypi.org/project/lute3/](https://pypi.org/pr
 
 > * Before starting, make sure you have python 3.8+ installed!  Run `python3 --version` from the terminal to check.
 
+* Create a Lute folder
```

Comment:
I see what you're doing here.  One thing I like about the current page is that it can be copied and pasted as-is in a console window, and it runs and you're done -- that's what's in the youtube intro video.  With your suggestion, each command becomes a separate thing, and I'd have to copy each block separately.

I can see how the current page would be overwhelming for non-technical reasons, and yours breaks it up.  I'm going to try some variations and see if I can have both of these available, somehow.